tools/tap-gtester

```diff
@@ -90,6 +90,8 @@ class GTestCompiler(NullCompiler):
                        print("ok %d %s" % (self.test_num, self.test_name))
                    if data == "FAIL":
                        print("not ok %d %s" % (self.test_num, self.test_name))
+                   if data == "SKIP":
+                       print("ok %d %s # skip" % (self.test_num, self.test_name))
```

Comment:
The code change makes sense. Just the commit message doesn't really fit for Cockpit, there is no test-pam.c .